lib/db/db_sqlite.rb

```diff
@@ -0,0 +1,85 @@
+require 'db/db'
+require 'sqlite3'
+
+class DBSQLite < DB
+
+  def initialize(sqlite, table, columns)
```

Comment:
I'll add another argument:
Who should care about table and column names? If the wrong ones are picked, ConMan is pretty useless/broken. And then details of the DB (table/column names) start leaking...